WasatchNET/Spectrometer.cs

```diff
@@ -1407,22 +1537,12 @@ virtual internal bool open()
 
             if (eeprom.hasCooling && degC != UNINITIALIZED_TEMPERATURE_DEG_C)
             {
-                // MZ: TEC doesn't do anything unless you give it a temperature first
+                // TEC doesn't do anything unless you give it a temperature first
                 logger.debug("setting TEC setpoint to {0} deg C", degC);
                 detectorTECSetpointDegC = degC;
 
-                // MZ: why don't we do this for ARM?  Is it automatic in FW?
-                if (!isARM)
-                {
-                    logger.debug("enabling detector TEC");
-                    detectorTECEnabled = true;
-                }
-            }
-
-            if (isSiG)
-            {
-                logger.debug("requiring throwaway after changing integration time");
-                throwawayAfterIntegrationTime = true;
+                logger.debug("enabling detector TEC");
+                detectorTECEnabled = true;

```

Comment:
What changed here?

Reply:
I moved "detectorTECEnabled = true" out of the "if (!isARM)" block, so that it would automatically enable the TEC on ARMs, which is what we're using for the multichannel system :-)